common/components/internal-header/_internal-header.scss

```diff
@@ -1,311 +1,182 @@
-$govuk-header-background: govuk-colour("black");
+$govuk-header-background: govuk-colour('black');
```

Comment:
Could we revert these quote changes please. I suspect it might be an IDE setting you have. Although we don't have some form of scss linting applied yet the standard approach up until now has been double quotes so it would be good to stick to that and reduce the noise in this PR.

Reply:
Ah right, I see. I think the intention of that definition is for `.js` files though as that's also a rule we use for JavaScript linting.

Currently almost all of our `.scss` files use double quotes. I might look to clarify this with either adding an extension override to the `.prettierrc` or adding style linting. 

But if we could revert it as part of this change for now that would keep it consistent with the rest of the files of the same type.